examples/src/examples/graphics/ground-fog/shader.frag

```diff
@@ -0,0 +1,37 @@
+
+            uniform sampler2D uTexture;
```

Comment:
Trim all the unnecessary whitespace in this file (and all other shader files that have the same problem).

Reply:
Yep working on it